pages/docs/data-ops/dbt-integration.mdx

```diff
@@ -0,0 +1,107 @@
+# dbt Integration
+
+## Overview
+
+Glean can import models directly from dbt. This is useful for incorporating Glean into existing DataOps workflows, allowing you to make changes to your dbt schema while identifying issues in downstream visualizations and dashboards.
+
+You must add the following fields to the `meta` key of each dbt model you want imported to Glean:
+
+```yaml
+# schema.yml
+
+version: 2
+
+models:
+  - # ... dbt model configuration 
+    meta:
+      glean:
+        glean: "1.0" # Glean dbt integration version
+        name: # Glean model name
+        source:
+          connectionName: # which Glean data connection to use
+    columns:
+      # ... your dbt columns
+```
+
+`source` has the same options as the corresponding field of [Glean data models](./config-schema/data-models.mdx). Unless otherwise specified, Glean automatically infers defaults for the source `physicalName` and `schema`.
```

Comment:
Maybe a phrasing that "you can remove the connection name if you just name the Glean database connection the same as the database name"